lib/src/components/list_item/notification_list_item.dart

```diff
@@ -0,0 +1,286 @@
+                  children: [
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Row(
+                          children: [
+                            if (!_notificationRead)
+                              ZetaIndicator(
+                                color: colors.blue,
+                                size: ZetaWidgetSize.small,
+                              ),
+                            Text(
+                              widget.title,
+                              style: const TextStyle(fontWeight: FontWeight.w500),
+                            ),
+                          ],
+                        ),
+                        Row(
+                          children: [
+                            if (widget.notificationTime != null) Text(widget.notificationTime!),
+                            Container(
+                              padding: const EdgeInsets.all(ZetaSpacing.x0_5),
+                              decoration: BoxDecoration(color: colors.negative, borderRadius: ZetaRadius.full),
+                              child: Icon(
+                                ZetaIcons.important_notification_round,
+                                color: colors.white,
+                                size: ZetaSpacing.x3,
+                              ),
+                            )
+                          ].gap(ZetaSpacing.x1),
+                        )
+                      ],
+                    ),
+                    SizedBox(
+                      height: ZetaSpacing.x11,
+                      child: Text(
+                        widget.message,
+                        maxLines: 2,
+                        overflow: TextOverflow.ellipsis,
+                      ),
+                    ),
+                  ].gap(ZetaSpacing.x1),
+                ),
+              ),
+            ].gap(ZetaSpacing.x2),
+          ),
+          if (widget.linkText != null)
+            Padding(
+              padding: const EdgeInsets.fromLTRB(56, 0, 0, 0),
+              child: InkWell(
+                onTap: () {
+                  setState(() {
+                    _notificationRead = true;
+                  });
+                  widget.linkOnClick!.call();
+                },
+                child: Row(
+                  children: [
+                    Icon(
+                      ZetaIcons.attachment_round,
+                      size: ZetaSpacing.x3,
+                      color: colors.primary,
+                    ),
+                    Text(
+                      widget.linkText!,
+                      style: TextStyle(fontSize: ZetaSpacing.x3, color: colors.primary),
+                    )
+                  ],
+                ),
+              ),
+            )
+          else
+            const SizedBox.square(
+              dimension: ZetaSpacing.x3,
+            ),
+          Container(
```

Comment:
Here, instead of hard coding a button you should just make the component accept an `action` argument which is of type `Widget`, that way the developer can put any action they need here.